### DISCUSSION TITLE
Why is there no action request ? And why can't I change the headers in the response action?

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#13370] [Feature Requests]
<img width="757" alt="image" src="https://user-images.githubusercontent.com/101538735/168953026-364e4b10-6f61-4ac7-9a98-ed6a1306bb0a.png">

<img width="766" alt="image" src="https://user-images.githubusercontent.com/101538735/168953355-f07b560c-5b31-45ea-b0f7-678042140642.png">

**Comment by azrikahar** [comment]
"action" happens at the end of a particular process/request, hence there is a `response` action (since it's finished, and Directus has responded) but not an actual request action.

There is the Init hook which should allow you to add a middleware for example to augment the response. Perhaps it would help others to get a better picture as well if you can share the end goal you are looking to achieve.
